SdfLibConfig.cmake.in

```diff
@@ -0,0 +1,15 @@
+# Include dependency macro support
+include(CMakeFindDependencyMacro)
+
+if(@SDFLIB_USE_OPENMP@)
+    find_package(OpenMP REQUIRED)
+    find_package(Threads REQUIRED)
+endif()
```

Comment:
```suggestion
if("@SDFLIB_USE_OPENMP@")
    find_dependency(OpenMP)
    find_dependency(Threads)
endif()
```
BTW the same pattern would work for every other dependency, instead of collecting them in `SDFLIB_DEPENDENCIES`.

Reply:
I already did with find_dependency and you say it will not work.
https://github.com/UPC-ViRVIG/SdfLib/pull/15#discussion_r1921148070

Reply:
You can refactor as you wish, after my PR.

Reply:
The point was that `CONFIG` doesn't work.